package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "terriajs",
-  "version": "7.10.0-mobx-33",
+  "version": "7.10.0-mobx-34",
```

Comment:
Remove this - release manager will bump versions

Reply:
done